Reneiski_Homework1/Reneiski_Homework1/Reneiski_Homework1.csproj.user

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I don't think you need to commit and merge this file

Reply:
Fixed